pinc/TallyBoard.inc

```diff
@@ -240,60 +240,53 @@ class TallyBoard
      * Parameters after `$radius` are an optimization for extracting
      * other info from the database at the same time.
      */
-    public function get_neighborhood(
-        $target_holder_id,
-        $radius,
-        $other_table,
-        $holder_id_expr,
-        $other_select_exprs,
-        $tally_alias,
-        $rank_alias
-    ) {
+    public function get_neighborhood($target_holder_id, $radius)
+    {
         assert($radius >= 0);
 
         $sql = sprintf(
             "
             SELECT
-                $holder_id_expr AS _holder_id,
-                IFNULL(tally_value,0) AS $tally_alias,
-                $other_select_exprs
+                IFNULL(tally_value, 0) AS current_tally,
+                u_id, username, u_privacy, date_created
             FROM
-                $other_table
-                LEFT OUTER JOIN current_tallies
-                ON (
-                    tally_name      = '%s'
-                    AND holder_type = '%s'
-                    AND holder_id   = $holder_id_expr
-                )
-            WHERE tally_value > 0 OR $holder_id_expr='$target_holder_id'
-            ORDER BY tally_value DESC, holder_id ASC
+                current_tallies
+                RIGHT OUTER JOIN users ON u_id = holder_id
+            WHERE
+                tally_name = '%s'
+                AND holder_type = '%s'
+                AND tally_value > 0
+            ORDER BY tally_value DESC, u_id ASC
             ",
             DPDatabase::escape($this->tally_name),
-            DPDatabase::escape($this->holder_type)
+            DPDatabase::escape($this->holder_type),
+            $target_holder_id
```

Comment:
is this used in the query? I only see 2 placeholders in the string, but I could be missing it.

Reply:
Good eyes! It is not and was left over from a prior iteration. I've pushed up a commit to remove it. 🙏